plover/messages/nl/LC_MESSAGES/plover.po

```diff
@@ -0,0 +1,1021 @@
+#: plover/gui_qt/dictionaries_widget_ui.py:146
+msgid "Move selected dictionaries down."
+msgstr "Beweeg de geselecteerde woordenboeken omlaag."
+
+#. Widget: “DictionaryEditor”.
+#: plover/gui_qt/dictionary_editor.py:163
+msgid "Strokes"
+msgstr "Aanslagen"
+
+#. Widget: “DictionaryEditor”.
+#: plover/gui_qt/dictionary_editor.py:166
+msgid "Translation"
+msgstr "Vertaling"
+
+#. Widget: “DictionaryEditor”.
+#: plover/gui_qt/dictionary_editor.py:169
+msgid "Dictionary"
+msgstr "Woordenboek"
+
+#. Widget: “DictionaryEditor”, windowtitle.
+#: plover/gui_qt/dictionary_editor_ui.py:100
+msgid "Plover: Dictionary Editor"
+msgstr "Plover: Bewerk woordenboek"
+
+#. Widget: “DictionaryEditor”, title.
+#: plover/gui_qt/dictionary_editor_ui.py:102
+msgid "Filter"
+msgstr "Filter"
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:104
+msgid "By strokes:"
+msgstr "Op aanslagen:"
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:106
+msgid "Apply"
+msgstr "Toepassen"
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:108
+msgid "By translation:"
+msgstr "Op vertaling:"
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:110
+msgid "Clear"
+msgstr "Leegmaken"
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:112
+msgid "&Delete"
+msgstr "&Verwijderen"
+
+#. Widget: “DictionaryEditor”, tooltip.
+#: plover/gui_qt/dictionary_editor_ui.py:114
+msgid "Delete selected entries."
+msgstr "Verwijder de geselecteerde regels."
+
+#. Widget: “DictionaryEditor”, tooltip.
+#: plover/gui_qt/dictionary_editor_ui.py:120
+msgid "Undo last add/delete/edit operation."
+msgstr "Maak de laatste toevoeg-/verwijder-/bewerk-operatie ongedaan."
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:124
+msgid "&New translation"
+msgstr "&Nieuwe vertaling"
+
+#. Widget: “LookupDialog”, tooltip.
+#: plover/gui_qt/lookup_dialog.py:13
+msgid "Search the dictionary for translations."
+msgstr "Zoek in het woordenboek naar vertalingen."
+
+#: plover/gui_qt/lookup_dialog.py:15
+msgid "Lookup"
+msgstr "Opzoeken"
+
+#. Widget: “LookupDialog”, windowtitle.
+#: plover/gui_qt/lookup_dialog_ui.py:56
+msgid "Plover: Dictionary Lookup"
+msgstr "Plover: Zoek in woordenboek"
+
+#: plover/gui_qt/machine_options.py:104
+msgid ""
+"Arpeggiate allows using non-NKRO keyboards.\n"
+"\n"
+"Each key can be pressed separately and the\n"
+"space bar is pressed to send the stroke."
+msgstr ""
+"Door te arpeggiëren is het mogelijk om een\n"
+"niet-NKRO-toetsenbord te gebruiken.\n"
+"\n"
+"Iedere toets kan één voor één ingedrukt worden,\n"
+"waarna de spatiebalk de aanslag verzendt."
+
+#. Widget: “MainWindow”, tooltip.
+#: plover/gui_qt/main_window_ui.py:172 plover/gui_qt/main_window_ui.py:212
+msgid "Disconnect and reconnect the machine."
+msgstr "Verbreek en herstel vervolgens de verbinding met de machine."
```

Comment:
"vervolgens" lijkt een beetje overbodig

Reply:
Yeah, this one is slightly awkward. Removed "vervolgens".